docker-compose.test.yaml

```
@@ -0,0 +1,95 @@
+        condition: service_healthy
+  user-test-db:
+    image: postgres:17-alpine
+    container_name: user-test-db
+    ports:
+      - "15433:5432"
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+      - POSTGRES_DB=user-test-db
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -U postgres" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)